src/SourceBuild/content/eng/sdkArchiveDiffs.targets

```diff
@@ -37,16 +36,20 @@
         <_changedFiles Include="@(_ContentDifferences)" Condition="'%(_contentDifferences.Kind)' != 'Unchanged'" />
       </ItemGroup>
 
+      <PropertyGroup>
+        <SdkArchiveDiffsReport>$(ArtifactsLogDir)\SdkArchiveDiffs\SdkArchiveDiffs.csv</SdkArchiveDiffsReport>
+      </PropertyGroup>
+
+      <WriteLinesToFile File="$(SdkArchiveDiffsReport)" Lines="%(_ContentDifference.Kind), %(_ContentDifferences.Identity)" />
+
       <Message Text="Difference in sdk archive: %(_changedFiles.Kind): %(_changedFiles.Identity)"
                Importance="High"
                Condition="'@(_changedFiles->Count())' != '0'"/>
       <Message Text="No differences in sdk archive file contents"
                Importance="High"
                Condition="'@(_changedFiles->Count())' == '0'" />
 
-      <Delete Files="$(_ClosestOfficialSdkPath)"
-              Condition="'$(_ClosestOfficialSdkPath)' == ''"/>
-
+      <Delete Files="$(_ClosestOfficialSdkPath)" />
```

Comment:
What's the value of `_ClosestOfficialSdkPath` and why do we need to delete the file?

Reply:
It's the downloaded sdk that we use as a baseline in the comparison. Maybe it should have a clearer name?

Reply:
Where does that get downloaded to? Which component decides where to download that file to?

Reply:
The `GetClosestSdk` task downloads it to a temp folder that it generates.

Reply:
Do we usually delete files that we put into a temporary directory? Just wondering.

Reply:
I'm not sure if we usually do or not

Reply:
OK disregard. I think the deletion is fine.